godtools/App/Services/Articles/Repository/Downloader/ArticleAemDownloadOperation.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import RequestOperation
```

Comment:
@levieggertcru why do you need the import if now code is added?

Reply:
Hey @gyasistory that code is now in a shared framework (pulled in via swift package manager) here -> https://github.com/CruGlobal/request-operation-ios.